content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -205,8 +205,10 @@ by the Kubernetes cluster, and then present that certificate to the Kubernetes A
 ### Create private key
 
```

Comment:
Perhaps it's helpful to explain the heritage of CN and O (they come from X.500 - see https://www.ibm.com/docs/en/zos/2.2.0?topic=SSLTBW_2.2.0/com.ibm.tcp.ipsec.ipsec.help.doc/com/ibm/tcp/ipsec/nss/NssImageServerPs.RB_X500.htm)

In X.500, I might be `commonName=Tim Bannister` and `o=The Scale Factory`. In a cluster I might be user `sftim` and group `acme-developers`.
Kubernetes could _in theory_ have used its own OID, or an existing schema, but does not. Right now the docs avoid explaining this gap.

You also, as far as I know, can't create a cert that assigns a user to multiple groups. If you can, we also don't explain that.

We don't have to fix everything in this PR. I do want us to recognise which gaps we're leaving unfilled if / when we merge this.

Reply:
A certificate can have [multiple organizations](https://github.com/golang/go/blob/8edcdddb23c6d3f786b465c43b49e8d9a0015082/src/crypto/x509/pkix/pkix.go#L124) specified.  This will assign the user to multiple groups.

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/authentication/#x509-client-certs talks about multi-group certificates:

> As of Kubernetes 1.4, client certificates can also indicate a user's group memberships using the certificate's organization fields. To include multiple group memberships for a user, include multiple organization fields in the certificate.
> 
> For example, using the openssl command line tool to generate a certificate signing request:
> 
> `openssl req -new -key jbeda.pem -out jbeda-csr.pem -subj "/CN=jbeda/O=app1/O=app2"`
>
> This would create a CSR for the username "jbeda", belonging to two groups, "app1" and "app2".